frontend/src/ches/MergeForm.jsx

```diff
@@ -524,16 +710,16 @@ class MergeForm extends Component {
                               onDrop={this.onExcelFileDrop}
                               accept="text/csv,.xls,application/msexcel,application/excel,application/vnd.ms-excel,application/vnd.openxmlformats-officedocument.spreadsheetml.sheet">
                               {({getRootProps, getInputProps}) => (
-                                <div {...getRootProps({className: 'dropzone'})}>
+                                <div {...getRootProps({className: 'dropzone-excel'})}>
                                   <input type="file" multiple {...getInputProps({className: 'dropzone-fileinput'})} />
-                                  <i className="m-sm-auto fas fa-2x fa-file-excel upload-icon" alt="upload xlsx"></i>
+                                  <i className="m-sm-auto fas fa-2x  fa-file-excel upload-icon" alt="upload xlsx"></i>
```

Comment:
Should remove the extra whitespace added

Reply:
addressed in next commit.